cmd/api/src/model/filter.go

```diff
@@ -96,8 +97,46 @@ type QueryParameterFilter struct {
 }
 
 type QueryParameterFilters []QueryParameterFilter
+
+
+func (f QueryParameterFilter) BuildGDBNodeFilter() graph.Criteria {
+    propertyRef := query.NodeProperty(f.Name)
+    value := guessFilterValueType(f.Value)
```

Comment:
We try to use `s` for receiver names so please update `f` to `s` here. In addition, it would be good to wrap these declarations with a `var` so they format nicely.